### ISSUE TITLE
app bit the dust or development dead?

### ISSUE THREAD

**Issue by Unixware** [openkaios/openkaios-store-client#4]
All I am getting is a "get data failed"

**Comment by sibeliusg** [comment]
Same for me. But it is also the same on B-hackers-Store client. maybe it is related to this: https://github.com/strukturart/bHacker-store-client/issues/71

**Comment by Unixware** [comment]
yes thats weird... maybe the problem is with the b-hackers server...

**Comment by dcowan-london** [comment]
Hey,
The reason this error is appearing is because the application is expecting the store version (from the server - https://banana-hackers.gitlab.io/store-db/data.json) to be 2 (https://github.com/strukturart/bHacker-store-client/blob/97a416036e154a558f73a3cabb1d70bf0c066150/application/assets/js/backend_api.js#L50) - and it seems to be 3.

**Comment by Unixware** [comment]
tested the version  2.1.5 and displays the same error....
